datanode/gateway/graphql/schema.graphql

```diff
@@ -623,6 +623,56 @@ type Query {
   "Query for historic ethereum key rotations"
   ethereumKeyRotations(nodeId: ID): EthereumKeyRotationsConnection!
 
+  "Estimate transaction fee"
+  estimateTransferFee(
+    "Sender's ID."
```

Comment:
It would be the sender's party ID, but I don't remember doing this change so I will have to check where it came from

Reply:
Oh, that's right, I moved the queries around so that they are in alphabetical order again, a lot of new API's were added out of order after we had ordered them for the mainnet release in August